test/extensions/filters/http/compressor/mock_compressor_library.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+
+package test.mock_compressor_library;
+
+message Unregistered {
```

Comment:
this is pretty nifty, I was struggling to get past the bootstrap exception to get to the unregistered exception. I'll add this to my test. Thanks for the inspiration.